onboarding.md

```diff
@@ -0,0 +1,27 @@
+# Seattle Flu Study onboarding
+
+## Adding new database users
+Before adding a new study member to the production database, confirm the DTUA is executed with Robin.
+
+Our convention for creating new usernames is first initial plus last name.
```

Comment:
I've actually been using the username folks have at their institution so it's the same as something they're already used to. This results in a mix of slightly different username formats.

Reply:
Oops, I didn't do that for the most recent user.